src/Spillgebees.Blazor.Map.Assets/src/styles.scss

```
@@ -257,14 +268,22 @@
 // maplibre resizes the canvas via its container ResizeObserver once the element fills the viewport
-.sgb-map-pseudo-fullscreen {
+.sgb-map-container.sgb-map-pseudo-fullscreen {
   position: fixed !important;
   inset: 0 !important;
   width: 100% !important;
   height: 100% !important;
   z-index: 99999;
 }
 
+.sgb-map-container:fullscreen,
+.sgb-map-container:-webkit-full-screen,
```

Stylelint (17.13.0)

[error] Vendor-prefixed selector ":-webkit-full-screen" (selector-no-vendor-prefix)

(selector-no-vendor-prefix)